### PR TITLE
[MM-18448] Prepare for 1.0 release

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Clean up, user experience changes, better icons

#### Links
[MM-18448](https://mattermost.atlassian.net/browse/MM-18448)

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -60,6 +62,11 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, commandArgs *model.CommandArg
 	if len(args) == 0 || args[0] != "/webex" {
 		return p.help(commandArgs), nil
 	}
+
+	if !p.getConfiguration().IsValid() {
+		return p.responsef(commandArgs, "The Webex plugin has not been configured correctly: the sitename has not been set. Please contact your system administrator."), nil
```
Comment (levb) [inline server/command.go]: nit: what is `sitename`? Is it a term?

File: server/store.go [hunk 2]
```diff
@@ -30,21 +30,16 @@ func hashkey(prefix, key string) string {
...
-		}
-		returnErr = errors.WithMessage(returnErr, "failed to get from store")
-	}()
+var ErrUserNotFound = errors.New("user not found")
 
+func (store store) get(key string, v interface{}) error {
 	data, appErr := store.plugin.API.KVGet(key)
 	if appErr != nil {
-		return appErr
+		return errors.New(appErr.Error())
```
Comment (levb) [inline server/store.go]: nit: why errors.New? for the callstack?

Comment (cpoile) [reply server/store.go]: @levb because it's returning an error, not an appErr

File: server/command.go (hunk 1 above)
Comment (cpoile) [reply server/command.go]: @levb I think that's the term that the slack addon used, and what Cisco uses... But it's confusing, I agree. I called the `teamsite.webex.com` in the settings the "Webex Site Hostname". Is there a proper term for the hostname+domainname combination?

File: server/store.go (hunk 2 above)
Comment (cpoile) [reply server/store.go]: Okay, fixed, thanks!

**Pull request merged**